app/controllers/application_controller.rb

```diff
@@ -1,2 +1,33 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::Base
+  def signin(user)
+    new_token = User.create_token
+    user.update_attributes(remember_token: new_token)
+    cookies.permanent[:remember_token] = new_token
+    retrive_user
```

Comment:
Why not just directly set the `current_user` with the `user`

Reply:
Done 👍